js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/AddEdgeModal.js

```diff
@@ -0,0 +1,140 @@
+/* global arangoHelper, $ */
```

Comment:
Are we still using G6?

Reply:
I need to display the functionality of my former G6 Graph Viewer implementation to "copy" the logic to the new vis.js Graph Viewer. All the G6 connections and naming will be removed in https://github.com/arangodb/arangodb/pull/18199.